api/api-version-v1/src/test/groovy/com/thoughtworks/go/apiv1/version/VersionRepresenterTest.groovy

```diff
@@ -0,0 +1,41 @@
+package com.thoughtworks.go.apiv1.version
+
+import com.thoughtworks.go.CurrentGoCDVersion
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+import static com.thoughtworks.go.api.base.JsonUtils.toObjectString
+import static net.javacrumbs.jsonunit.fluent.JsonFluentAssert.assertThatJson
+
+class VersionRepresenterTest {
+
+  private CurrentGoCDVersion currentGoCDVersion;
+
+  @BeforeEach
+  void setUp() {
+    currentGoCDVersion = new CurrentGoCDVersion()
```

Comment:
This field should be mocked. This is calling the real `CurrentGoCDVersion` object, which value will keep changing with every new commit.